### PR TITLE
Issue #317: Updated readme templates with more details. Resolves #317

### LINKED ISSUE #317: Provide better info in the following README templates
Currently, these are pretty minimal:

docs/topics/crud-README.adoc
docs/topics/configmap-README.adoc
docs/topics/health-check-README.adoc
docs/topics/circuit-breaker-README.adoc
docs/topics/rest-http-secured-README.adoc

Update with better details on FMP deploy and interaction

### PR DESCRIPTION
**Pull request by rhoads-zach**

### CODE REVIEW

**Comment by rhoads-zach** [comment]
merged: https://github.com/openshiftio/appdev-documentation/commit/55f0db9d46559f067c26a19122d682869c72a6c4

File: docs/topics/circuit-breaker-README.adoc [hunk 1]
```diff
@@ -6,24 +6,89 @@ ifeval::["{runtime}" == "Spring Boot"]
...
+
+{greeting-app-name}-MY_PROJECT_NAME.192.168.42.158.nip.io
+
+$ oc get route {name-app-name} -o jsonpath={$.spec.host}
+
+{name-app-name}-MY_PROJECT_NAME.192.168.42.158.nip.io
+----
+
+
+You can use the form from at browser at your application's url or you can use the `curl` command:
```
Comment (Ladicek) [inline docs/topics/circuit-breaker-README.adoc]: I'm pretty sure "form from at browser at your application's url" is a mistake, but I don't know what exactly is wrong :-)

File: docs/topics/configmap-README.adoc [hunk 2]
```diff
@@ -57,4 +62,111 @@ $ curl http://localhost:8080/api/greeting?name=Sarah
...
+
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get route {app-name} -o jsonpath={$.spec.host}
+
+{app-name}-MY_PROJECT_NAME.192.168.42.158.nip.io
+----
+
+
+You can use the form from at browser at your application's url or you can use the `curl` command:
```
Comment (Ladicek) [inline docs/topics/configmap-README.adoc]: Same as above.

File: docs/topics/crud-README.adoc [hunk 3]
```diff
@@ -28,3 +28,101 @@ endif::[]
...
+
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get route MY_APP_NAME -o jsonpath={$.spec.host}
+
+MY_APP_NAME-MY_PROJECT_NAME.192.168.42.158.nip.io
+----
+
+
+You can use the form from at browser at your application's url or you can use the `curl` command:
```
Comment (Ladicek) [inline docs/topics/crud-README.adoc]: Same as above.

File: docs/topics/health-check-README.adoc [hunk 4]
```diff
@@ -51,4 +56,54 @@ $ curl http://localhost:8080/api/greeting?name=Sarah
...
+
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get route {app-name} -o jsonpath={$.spec.host}
+
+{app-name}-MY_PROJECT_NAME.192.168.42.158.nip.io
+----
+
+
+You can use the form from at browser at your application's url or you can use the `curl` command:
```
Comment (Ladicek) [inline docs/topics/health-check-README.adoc]: Same as above.

File: docs/topics/rest-http-secured-README.adoc [hunk 5]
```diff
@@ -1,29 +1,108 @@
...
+Using auth server URL: {sso-app-url}
+Available application endpoint names: [{secured-app-name}]
+
+$ oc get route {secured-app-name} -o jsonpath={$.spec.host}
+
+{secured-app-name}-MY_PROJECT_NAME.192.168.42.158.nip.io
+----
+
+
+You can use the form from at browser at your application's url or you can use the provided client:
```
Comment (Ladicek) [inline docs/topics/rest-http-secured-README.adoc]: Same as above.

**Pull request closed without merging**